contents/docs/libraries/js/index.mdx

```diff
@@ -505,6 +505,7 @@ Some of the most relevant options are:
 | `mask_all_text`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                                       | Prevent PostHog autocapture from capturing any text from your elements.                                                        |
 | `mask_all_element_attributes`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                         | Prevent PostHog autocapture from capturing any attributes from your elements.                                                  |
 | `opt_out_capturing_by_default`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                        | Determines if users should be opted out of PostHog tracking by default, requiring additional logic to opt them into capturing by calling `posthog.opt_in_capturing`. |
+| `opt_out_persistence_by_default`<br/><br/>**Type:** Boolean<br/>**Default:** `false`                                                                                                        | Determines if users should be opted out of browser data storage by this PostHog instance by default. |
```

Comment:
I don't... I'll be lazy and tag @benjackwhite who I think has a good handle on JS SDK privacy functions (Ben, tell me not to be lazy and I'll figure it out if need be :))

Reply:
confirmed that calling `opt_in_capturing` is enough to revert `opt_out_persistence_by_default: true` since `opt_in_capturing` calls its internal methods with `enable_persistence=true` by default.